src/UI/Component/Modal/ReplaceContentSignal.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+use ILIAS\UI\Component\Signal;
+
+/**
+ * This signal replaces the content of a Modal with some content returned by an ajax request.
+ *
+ * @author Jesús Lópéz <lopez@leifos.com>
+ * @package ILIAS\UI\Component\Modal
+ */
+interface ReplaceContentSignal extends Signal
```

Comment:
This means e.g. moving
UI/Implementation/Component/Popover/ReplaceContentSignal.php
to
UI/Implementation/Component/Signals/ReplaceContentSignal.php
?

Reply:
Yes, but we would need to think about a good namespace then, I'm not sure if `UI\Implementation` is really correct. This could well be done after this PR, though.